bucket/ravendb.json

```diff
@@ -0,0 +1,55 @@
+{
+    "homepage": "https://ravendb.net",
+    "description": "NoSQL database built on .NET Core.",
+    "version": "4.2.4",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://ravendb.net/terms/commercial"
+    },
+    "suggest": {
+        "Visual C++ Redistributable 2015": "extras/vcredist2015"
+    },
+    "architecture": {
+        "32bit": {
+            "url": "https://daily-builds.s3.amazonaws.com/RavenDB-4.2.4-windows-x86.zip",
+            "hash": "e93516db0f8f37772de490068a655fc7b3794b6061b5c8abffaafa9dde4fd197"
+        },
+        "64bit": {
+            "url": "https://daily-builds.s3.amazonaws.com/RavenDB-4.2.4-windows-x64.zip",
+            "hash": "8e65f887a677e78a92e57faf71363c17a1b2bf54b0c490f5b2618b597117dd99"
+        }
+    },
+    "pre_install": "if(!(Test-Path \"$persist_dir\\Server\\settings.json\")) {Set-Content \"$dir\\Server\\settings.json\" -Value $null}",
```

Comment:
- Do not use $null in set-content
    - Use new-item instead
- Always use encoding while using set-content